extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -178,11 +294,25 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
   /**
    * @override
    */
-  closeInternal_() {
-    super.closeInternal_();
+  closeInternal_(shouldAnimate = true) {
+    if (this.state_ === DrawerState.CLOSED) {
+      return;
+    }
+
+    super.closeInternal_(shouldAnimate);
 
     this.storeService_.dispatch(Action.TOGGLE_SYSTEM_UI_IS_VISIBLE, true);
 
+    const storyEl = closest(this.element, el => el.tagName === 'AMP-STORY');
+    const animationEl = storyEl.querySelector(
+      '.i-amphtml-story-page-attachment-expand'
+    );
+    if (animationEl) {
+      this.mutateElement(() => {
+        removeElement(dev().assertElement(animationEl));
```

Comment:
did you consider hiding / removing the css class from the element instead of creating and removing the element on each open / close?

Reply:
Yeah that was my original implementation, but having the animation a direct child of `amp-story` made everything much easier (z-index stack issues). Because it's not really supposed to be there I wanted to remove it when it's not displayed